### ISSUE TITLE
add option for notify position

### ISSUE THREAD

**Issue by k0377** [BrowserSync/browser-sync#685]
sometimes it is annoying, if you work on a navigation top bar or a header, browsersync pushes its notification on the upper right corner. i would like to see an option to position the notification in the other corners as well.

`notify: topleft | topright | bottomleft | bottomright`

I know I can position the notification with css myself.

**Comment by shinnn** [comment]
Good idea, but I prefer user-editable CSS that would be more flexible.

**Comment by shakyShane** [comment]
You can override the defaults too, although you've have to provide the entire set.

This would set it to the bottom right

```js
notify: {
    styles:  [
        "display: none",
        "padding: 15px",
        "font-family: sans-serif",
        "position: fixed",
        "font-size: 0.9em",
        "z-index: 9999",
        "bottom: 0px",
        "right: 0px",
        "border-bottom-left-radius: 5px",
        "background-color: #1B2032",
        "margin: 0",
        "color: white",
        "text-align: center"
    ];
}
```

Ideally this would something you can tweak from the UI

**Comment by shakyShane** [comment]
https://github.com/BrowserSync/browser-sync-client/blob/master/lib/notify.js#L38-L40

**Comment by shinnn** [comment]
Oh I didn't aware of `options.notify.styles`.
